internal/backends/mcu/lib.rs

```diff
@@ -425,7 +430,7 @@ pub fn init_with_display<Display: Devices + 'static>(display: Display) {
     });
 }
 
-#[cfg(not(any(feature = "pico-st7789", feature = "simulator")))]
+#[cfg(not(any(feature = "pico-st7789", feature = "stm32h735g", feature = "simulator")))]
```

Comment:
This looks like dead code to me now. Or is there any way that this function will be included? :)

Reply:
This is included when no mcu backend is specified, and it just render a frame.
But yeah, we need to clean that up at some point. I would like to redo the backend API after your vacation.